config/schema/islandora_solr_metadata.data_types.schema.yml

```diff
@@ -0,0 +1,68 @@
+islandora_solr_metadata.config:
+  type: mapping
+  label: Metadata Display Config
+  mapping:
+    fields:
+      type: sequence
+      label: Display Fields
+      sequence:
+        type: islandora_solr_metadata.field
+    description:
+      type: mapping
+      label: Metadata Display Description
+      mapping:
+        description_field:
+          type: string
+          nullable: false
+        description_label:
+          type: label
+        truncation:
+          type: islandora_solr_metadata.truncation_options
+    cmodel_associations:
+      type: sequence
+      sequence:
+        type: string
+islandora_solr_metadata.field:
+  type: mapping
+  label: Metadata Display Field
+  mapping:
+    field:
+      type: string
+      nullable: false
+    weight:
+      type: integer
+    display_label:
+      type: label
+    hyperlink:
+      type: boolean
+    uri_replacement:
+      type: string
+      nullable: true
+    date_format:
+      type: string
```

Comment:
Reviewing the [available core types](https://www.drupal.org/docs/8/api/configuration-api/configuration-schemametadata)... would it make more sense to use `date_format`?